packages/web/app/src/lib/preflight/components/EditorTitle.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from 'react';
+import { cn } from '@/lib/utils';
+
+export function EditorTitle(props: { children: ReactNode; className?: string }) {
+  return (
+    <div className={cn('cursor-default text-base font-semibold tracking-tight', props.className)}>
+      {props.children}
+    </div>
+  );
+}
```

Comment:
Is it really necessary to have a file for these 10 lines?

Reply:
Maybe its time to add some rules.

Reply:
if it can be enforced via a linter then I think it'd be fine to move to a stricter separation of components by file.
But otherwise we can keep it loose b/c i dislike enforcing these sorts of things manually.

I agree that if things become unruly then split.